catkin_ws/src/simulation/worlds/auv.sdf

```diff
@@ -0,0 +1,83 @@
+<?xml version='1.0'?>
+<sdf version="1.4">
+        <static>false</static>
+
+        <model name="gate">
+            <static>true</static>
+            <link name="gate">
+                <pose>-1 0 -1 0 0 0</pose>
+                <visual name="gate_visual">
+                    <geometry>
+                        <mesh>
+                            <uri>../assets/quali/meshes/gate.dae</uri>
+                        </mesh>
+                    </geometry>
+                </visual>
+                <collision name="gate_visual">
+                    <geometry>
+                        <mesh>
+                            <uri>../assets/quali/meshes/gate.dae</uri>
+                        </mesh>
+                    </geometry>
+                </collision>
+            </link>
+        </model>
+
+          <model name="auv">
+            <static>false</static>
+            <link name="auv_link">
```

Comment:
if you have two geometries you could split them up (ie. auv_hull_link and auv_base_link) and then add a static joint between them - it might be easier for inertia calculations